src/Xrefcheck/CLI.hs

```diff
@@ -69,34 +70,34 @@ data Command
   | DumpConfig Flavor FilePath
 
 data Options = Options
-  { oConfigPath       :: Maybe FilePath
-  , oRoot             :: FilePath
-  , oMode             :: VerifyMode
-  , oVerbose          :: Bool
-  , oShowProgressBar  :: Maybe Bool
-  , oTraversalOptions :: TraversalOptions
-  , oVerifyOptions    :: VerifyOptions
+  { oConfigPath        :: Maybe FilePath
+  , oRoot              :: FilePath
+  , oMode              :: VerifyMode
+  , oVerbose           :: Bool
+  , oShowProgressBar   :: Maybe Bool
+  , oExclusionOptions  :: ExclusionOptions
+  , oNetworkingOptions :: NetworkingOptions
   }
 
-data TraversalOptions = TraversalOptions
+data ExclusionOptions = ExclusionOptions
   { toIgnored :: [RelGlobPattern]
```

Comment:
Since the data type has been renamed to `ExclusionOptions`, let's rename the fields' prefix to `eo`

```suggestion
  { eoIgnored :: [RelGlobPattern]
```